vue/sbc-common-components/src/components/BaseAddress.vue

```diff
@@ -160,52 +164,54 @@ export default class BaseAddress extends Mixins(ValidationMixin, CountriesProvin
   })
   readonly address: object
 
-  /**
-   * Indicates whether the address should be shown in editing mode (true) or display mode (false).
-   */
+  /** Whether the address should be shown in editing mode (true) or display mode (false). */
   @Prop({ default: false })
   readonly editing: boolean
 
-  /**
-   * The address schema containing Vuelidate rules.
-   */
+  /** The address schema containing Vuelidate rules. */
   @Prop({ default: null })
   readonly schema: any
 
-  /**
-   * A local (working) copy of the address, to contain the fields edited by the component (ie, the model).
-   */
-  private addressLocal: object = { }
+  /** A local (working) copy of the address, to contain the fields edited by the component (ie, the model). */
+  private addressLocal: object = {}
 
-  /**
-   * A local (working) copy of the address schema.
-   */
-  // TODO: this misses the initial country setting (re: on edit)
-  private schemaLocal: any = { ...this.schema }
+  /** A local (working) copy of the address schema. */
+  private schemaLocal: any = {}
 
-  /**
-   * Getter for Address Country, to simplify template and so we can watch it below.
-   */
+  /** A unique id for this instance of this component. */
+  private uniqueId = uniqueId()
+
+  /** A unique id for the Street Address input. */
+  private get streetAddressId (): string {
+    return `street-address-${this.uniqueId}`
```

Comment:
Off the top of your head, what is the difference between lodash's uniqueId and something like UUID?

Reply:
This lodash method returns a simple id starting from '1' and incrementing each time you call it.

The UUID package returns a random uuid string, which would have been total overkill here.

Reply:
Yea, the UUID string is pretty intense. Thanks for the info!